src/bcf/report/mod.rs

```diff
@@ -10,6 +12,7 @@ pub mod table_report;
 pub fn embed_js(
     output_path: &str,
     custom_table_report_js: Option<&str>,
+    custom_js_files: Option<Values>,
```

Comment:
Here, a Vec is sufficient. No need to wrap the values in an option. If the vec is empty, the loop below will not be entered.

Reply:
Oh that is pretty smart, I didn't think of that thanks!